tests/test_backend.py

```diff
@@ -1338,6 +1476,42 @@ def test_summary_logging(self):
             # The last message should be the summary output
             self.assertEqual(cm.output[-1], SUMMARY_LOG_REPORT)
 
+    def test_blacklist_ids(self):
+        """Test whether the blacklist attribute is correctly added"""
```

Comment:
It is more clear to say that "Test whether blacklisted items are not returned"""